### PR TITLE
add missing UI for specifying alternative JRE path to Kotlin run conf…

### PR DESCRIPTION
**Pull request by yole**
…iguration

### CODE REVIEW

File: idea/src/org/jetbrains/kotlin/idea/run/JetRunConfigurationEditor.java [hunk 1]
```diff
@@ -61,6 +64,8 @@ protected void applyEditorTo(JetRunConfiguration configuration) throws Configura
         myModuleSelector.applyTo(configuration);
         myCommonProgramParameters.applyTo(configuration);
         configuration.setRunClass(myMainClassField.getText());
+        configuration.setAlternativeJrePath(alternativeJREPanel.getPath());
+        configuration.setAlternativeJrePathEnabled(alternativeJREPanel.isPathEnabled());
```
Comment (cy6erGn0m) [inline idea/src/org/jetbrains/kotlin/idea/run/JetRunConfigurationEditor.java]: Has it real effect during run? Is it enough to just set it to a configuration?

Comment (yole) [inline idea/src/org/jetbrains/kotlin/idea/run/JetRunConfigurationEditor.java]: Yes. JetRunConfiguration already implements CommonJavaRunConfigurationParameters and exposes those values through that API to the shared code in IDEA.

**Comment by NataliaUkhorskaya** [comment]
Java has a warning if the path to jre is not valid, seems that Kotlin should have it too.

**Comment by yole** [comment]
Added an implementation of checkConfiguration() to JetRunConfiguration.

**Comment by NataliaUkhorskaya** [comment]
Shouldn't we align all lines by vertical?
untitled333

**Comment by yole** [comment]
Added alignment.

**Pull request merged**